### ISSUE TITLE
Should we pin other CUDA components tightly?

### ISSUE THREAD

**Issue by leofang** [conda-forge/libcusparse-feedstock#4]
During the review in staged-recipes, there was [a discussion](https://github.com/conda-forge/staged-recipes/pull/21924#discussion_r1155112007) on whether this pinning should/can be relaxed, given that we already pin `cuda-version` tightly:
https://github.com/conda-forge/libcusparse-feedstock/blob/38c8582741c3f03b2372b7f6371de081f724e0ab/recipe/meta.yaml#L55
Opening an issue here to track the discussion.

**Comment by jakirkham** [comment]
Discussion happening in issue ( https://github.com/conda-forge/libcusolver-feedstock/issues/5 )

**Comment by bdice** [comment]
I would agree to relax this in favor of only a pinning on `cuda-version`, but the status quo is fine for now. (It's easy to relax pinnings and hard to make them stricter.) As far as I can tell, the current pinning is no stricter than what `cuda-version` would enforce.

**Comment by jakirkham** [comment]
It appears in some cases users want `run_exports` (and even have added them in some cases). For example, `cuda-cupti-dev` had a `run_exports` added in PR ( https://github.com/conda-forge/cuda-cupti-feedstock/pull/10 ) and requested it be backported ( https://github.com/conda-forge/cuda-cupti-feedstock/issues/14 ). So these do seem useful for users

As to how strict the pinning should be, don't have strong feelings here

Re-reading the linked thread it looks like @kkraus14 and you had different perspectives. Maybe it is worth discussing them in this context (as this may be different than the general pinning discussion in that thread). Once a resolution is reached we can look at implementation

**Comment by sisodia1701** [comment]
Moving to backlog as needs further discussion.
